packages/components/molecules/f-skeleton-loader/src/assets/scss/common.scss

```diff
@@ -8,3 +8,33 @@ $theme: 'jet' !default;
 $skeleton-loader-heading-height: spacing(x4);
 $skeleton-loader-text-height: spacing(x2);
 $skeleton-loader-bone-background-color: $grey--mid;
+$skeleton-loader-image-height: spacing(x8);
+
+@keyframes loading {
+    100% {
+        transform: translateX(100%);
+    }
+}
+
+.c-text,
```

Comment:
Will we need to be careful these names don't clash, or do they get mangled in css modules?

Reply:
They all get mangled in the css module:

`class="Image_c-image_CmSiT RestaurantCarouselCard_c-restaurantCarouselCard-background_xC_PR"`

`animation: 1.5s ease 0s infinite normal none running Image_loading_JDTOd;`